requirements.txt

```diff
@@ -9,16 +9,21 @@ fuzzywuzzy==0.6.1
 graphene==0.10.2
 jsonschema==2.5.1
 lxml==3.8.0
-PyYAML==3.12
 requests>=2.5.2,<3.0.0
+PyYAML==3.11
 setuptools==36.3.0
 simplejson==3.8.1
 sqlalchemy==0.9.9
 -e git+https://git@github.com/NCI-GDC/signpost.git@c5d499936943e71eefe2ec4b3d4ced6ac48f35c0#egg=signpost
--e git+https://git@github.com/NCI-GDC/psqlgraph.git@1.2.0#egg=psqlgraph
+#-e git+https://git@github.com/NCI-GDC/psqlgraph.git@1.2.0#egg=psqlgraph
+-e git+https://git@github.com/NCI-GDC/psqlgraph.git@54c9f58c38ce321757ca28cfe3a42994c87719bd#egg=psqlgraph
 -e git+https://git@github.com/uc-cdis/cdis-python-utils.git@0.1.5#egg=cdispyutils
 -e git+https://git@github.com/uc-cdis/userdatamodel.git@f25f5dfb356e80c45153ffd3e91b399be672a81c#egg=userdatamodel
 -e git+https://git@github.com/uc-cdis/cdis_oauth2client.git@0.1.1#egg=cdis_oauth2client
 -e git+https://git@github.com/uc-cdis/indexclient.git@d49134f4626b69a8ef02c189ed0047ad1a635cb0#egg=indexclient
 -e git+https://git@github.com/uc-cdis/dictionaryutils.git@1.1.0#egg=dictionaryutils
 -e git+https://git@github.com/uc-cdis/datamodelutils.git@0.2.0#egg=datamodelutils
+# required for gdcdatamodel, not required for sheepdog
+-e git+https://git@github.com/NCI-GDC/cdisutils.git@4a75cc05c7ba2174e70cca9c9ea7e93947f7a868#egg=cdisutils
+-e git+https://git@github.com/NCI-GDC/gdcdatamodel.git@1.1.0#egg=gdcdatamodel
```

Comment:
Consider switching this to https://github.com/NCI-GDC/gdcdatamodel/commit/ec25990ed1e5a62984cee91f4fc8eb3cdbedf970 for now since we know that branch is necessary for some commons.

Reply:
Fixed in https://github.com/uc-cdis/sheepdog/pull/32/commits/f5c67f17c1caa282390884aeeacdc55c8394ec07. Thanks